gpt4all-chat/src/chatmodel.h

```diff
@@ -45,7 +45,7 @@ struct PromptAttachment {
         buffer.open(QIODevice::ReadOnly);
         const QString md = XLSXToMD::toMarkdown(&buffer);
         buffer.close();
-        return md;
+        return u"## Attached: %1\n\n%2"_s.arg(file(), md);
```

Comment:
I don't think sheets and files should have the same heading size, since sheets are logically part of files/workbooks. Probably the sheet names should be H3 (`### Sheet 1`).